src/components/CollectionBox.tsx

```diff
@@ -44,9 +43,8 @@ export const CollectionBox: React.FC<CollectionBoxProps> = ({collectionName, col
           }
           {!isEditing && (
               <>
-            <Link to={`/collection/${collectionId}`}>
```

Comment:
Why was this removed? How will the user get inside the collection?

Reply:
That piece is actually happening now in PageLayoutComponent.tsx at about line 121